content/articles/quarkus-configuration-profiles/index.md

```diff
@@ -0,0 +1,576 @@
+  - [Setting up variables](#setting-up-variables)
+  - [Injecting the variables](#injecting-the-variables)
+  - [Testing the variables](#testing-the-variables)
+    - [Testing in the dev profile](#testing-in-the-dev-profile)
+    - [Testing in the prototype profile](#testing-in-the-prototype-profile)
+- [Configuration profiles in YAML files](#configuration-profiles-in-yaml-files)
+- [Conclusion](#conclusion)
+- [References](#references)
+
+### Key takeaways
+Some points to be taken in by the end of the article follow up include:
+- What is Quarkus Configuration Profiles
+- Importance of configuration profiles
+- Some ways of setting up configuration profiles in Quarkus. These include setting the profiles in the `application.properties` and 'YAML' files.
+- Easily navigating between the profiles set during the development stages.
+
+### Prerequisites
+For one to follow up with the code examples and explanations given in the article, the following will be required:
+- Java Development Kit installed on the machine. The recommendation is the latest JDK in the market. JDK 17 is used for the article.
+- A good Java IDE. The recommendation is the latest version of the IntelliJ Ultimate edition. This choice is because it supports MicroProfile, among which Quarkus is one of them. IntelliJ version `2021.2.2` was used for the article, as seen in the screenshot captured.
+- Familiar with databases.
+- A good internet connection. The internet is for fetching indexes for quick development. It will also be used in the generation of the project.
+
+> **NOTE**:- These pre-requisites may change as time goes by ever since the day that the article is published.
+> Make sure to follow up on _the latest versions_ of the technologies mentioned above.
+> The appearance of the applications used may also differ based on the release dates.
+
+### What is Quarkus Configuration Profiles?
+
+Quarkus, like any other MicroProfile, follows the MicroProfile specifications. This property makes it to be able to allow developers to set the variables found in the whole project seamlessly so that they can be accessed in the project scope during different project life cycles.
+The settings of the project are usually stored in the `application.properties` file. The file can be located in the `resources` folder found in the `main` folder. Some variables that can be set include pointing out to **Datasource** such as _postgresql_ or _mysql_, **Datasource properties** and settings such as the _username_, _password_, _url_ and _access port_ among others.
+
+Configuration files can hold variables needed during the operation of the application. These can include variables that are to be isolated from the main application source code files. The variables can later be injected into the application when needed only.
+Developers always prefer to use the config files to set the overall system properties and environment variables.
+
```

Comment:
This paragraph also misses the transition between sentences!

Reply:
Done